service/tbc/tbc.go

```diff
@@ -146,6 +182,11 @@ func sliceChainHash(ch chainhash.Hash) []byte {
 	return ch[:]
 }
 
+type Seed struct {
+	Host string
+	Port string
+}
```

Comment:
I think instead of using a struct like this, we could keep the host and port in one string, e.g. "localhost:43111" and split it using `net.SplitHostPort`: https://pkg.go.dev/net#SplitHostPort

This would make the seeds easier to pass, and makes the configuration variable that needs to be added to `cmd/tbcd` easier to work with.

What do you think about this?

Reply:
Yes.

Reply:
Yeah I was on the fence about doing that here or at the consumer (op-geth, for example) level. I can change it here 👍